internal/controller/pod_webhook.go

```diff
@@ -121,3 +125,33 @@ func hasMIGResource(pod *v1.Pod) bool {
 	}
 	return false
 }
+
+func performQuotaArithmetic(pod *v1.Pod, req admission.Request) admission.Response {
+	// assumption is that workloads will have 1 container where
+	// MIG is requested.
+	for _, container := range pod.Spec.Containers {
```

Comment:
Can you add a TODO here to note that instead of only iterating over regular containers, we should also consider other types of containers (such as init containers) in future?